src/os/bluestore/BlueStore.cc

```diff
@@ -6909,8 +6909,11 @@ int BlueStore::_open_super_meta()
       ::decode(bluefs_extents, p);
     }
     catch (buffer::error& e) {
-      derr << __func__ << " unable to read bluefs_extents" << dendl;
-      return -EIO;
+      string kv_backend = cct->_conf->bluestore_kvbackend;
```

Comment:
I got it.
I think what you said is better.
I'll push it, reflecting on your comment.